src/main/java/com/minecolonies/coremod/lib/Constants.java

```diff
@@ -1,13 +1,16 @@
-package com.minecolonies.lib;
+package com.minecolonies.coremod.lib;
 
+/**
+ * Some constants needed for the whole mod.
+ */
 public class Constants
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1118)